portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,22 +27,48 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<Comment> comments = new ArrayList<>();
+
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     
-    //Create a list of comments
-    ArrayList<String> comments = new ArrayList<String>(3);
-    comments.add("What other anime have you watched?");
-    comments.add("How old are your nephews?");
-    comments.add("How do you like U of T?");
-
     // Convert comments list to JSON
     Gson gson = new Gson();
     String json = gson.toJson(comments);
 
     // Send json to server
     response.setContentType("applications/json;");
     response.getWriter().println(json);
+    System.out.println("Sent json to client.");
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get input from the form.
+    String name = getParameter(request, "name", "Anonymous");
+    String message = getParameter(request, "message", "No comment.");
+    System.out.println("Retireved input from form.");
```

Comment:
If the print statements are for debugging purpose, I would recommend adding a TODO to remove them later. Else, if you are using them logging purpose - use Logger.log instead.